<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-D36JS0L" ignored="0" name="ModulerFramework" tests="3" failures="1" timestamp="12 Sep 2018 12:31:21 GMT" time="0.061" errors="0">
  <testcase name="preconditions" time="0.002" classname="testNGDemo.TestSkipDemo">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at testNGDemo.TestSkipDemo.preconditions(TestSkipDemo.java:18)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- preconditions -->
  <testcase name="tearDown" time="0.0" classname="testNGDemo.TestSkipDemo"/>
  <testcase name="testCode" time="0.0" classname="testNGDemo.TestSkipDemo">
    <skipped/>
  </testcase> <!-- testCode -->
</testsuite> <!-- ModulerFramework -->
